CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+# Contributing Guidelines
+
+The following is a set of guidelines for contributing to the NGINX AWS Auto-Scale Group Sync tool. We really appreciate that you are considering contributing!
+
+#### Table Of Contents
+
+[Ask a Question](#ask-a-question)
+
+[Getting Started](#getting-started)
+
+[Contributing](#contributing)
+
+[Style Guides](#style-guides)
+  * [Git Style Guide](#git-style-guide)
+  * [Go Style Guide](#go-style-guide)
+
+[Code of Conduct](#code-of-conduct)
+
+## Ask a Question
+
+We will have a public forum soon where you can come and ask questions and have a discussion. For now please open an Issue on GitHub with the label `question`.
+
+
+## Getting Started
+
+Read the installation, configuration and building steps in the [README](https://github.com/nginxinc/nginx-asg-sync/blob/master/README.md).
+
+### Project Structure
+
+* nginx-asg-sync is a service written in Go that works with NGINX Plus.
+* The main code is found under `/cmd/sync/`
+* Tools for building the service for supported Operating Systems are found under `/build/package`
+* The project dependencies reside in the `/vendor`. We use [dep](https://github.com/golang/dep) for managing dependencies.
+* There is a Makefile at the project root used in the build steps.
+
+## Contributing
+
+### Report a Bug
+
+To report a bug, open an issue on GitHub with the label `bug` using the available bug report issue template. Please ensure the issue has not already been reported.
+
+### Suggest an Enhancement
+
+To suggest an enhancement, please create an issue on GitHub with the label `enhancement` using the available feature issue template.
+
+### Open a Pull Request
+
+* Fork the repo, create a branch, submit a PR when your changes are tested and ready for review
+* Fill in [our pull request template](https://github.com/nginxinc/nginx-asg-sync/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+
+Note: if you’d like to implement a new feature, please consider creating a feature request issue first to start a discussion about the feature.
+
+## Style Guides
+
+### Git Style Guide
+
+* Keep a clean, concise and meaningful git commit history on your branch, rebasing locally and squashing before submitting a PR
+* Follow the guidelines of writing a good commit message as described here https://chris.beams.io/posts/git-commit/ and summarised in the next few points
```

Comment:
`here https://chris.beams.io/posts/git-commit/ and summarised in the next few points` -> `[here] (https://chris.beams.io/posts/git-commit/ and summarised in the next few points)`

Reply:
thanks @Dean-Coakley but I want the link to appear as it is